docker-compose.yml

```diff
@@ -22,6 +26,7 @@ services:
       DATABASE_HOST: db
       BROKER_URL: redis://redis:6379/4
```

Comment:
I think you changed this to `REDIS_URL` in tox.ini, you'll need to make the same change here